apps/api/lib/validations/user.ts

```diff
@@ -104,6 +104,7 @@ const schemaUserEditParams = z.object({
     .optional()
     .nullable(),
   locale: z.nativeEnum(locales).optional().nullable(),
+  avatar: z.string().optional(),
```

Comment:
We should add validation that this string is actually a base64 string.

Reply:
@keithwillcode, Could you please check the new changes?

I separated the `isValidBase64` method in utils files, so it could be used for both images and texts, I don't know if I am missing something. hopefully not :)